### ISSUE TITLE
Create release to add to sacrebleu

### ISSUE THREAD

**Issue by mjpost** [wmt-conference/wmt24-news-systems#3]
Hi, can someone create a release of this repo? It should be sufficient to add a tag. This will then let me add it as a download target. For example, here is the URL for wmt23: https://github.com/wmt-conference/wmt23-news-systems/archive/refs/tags/v.0.1.tar.gz.

**Comment by mjpost** [comment]
CC: @zouharvi @kocmitom

**Comment by zouharvi** [comment]
Hi Matt, does this suffice?

https://github.com/wmt-conference/wmt24-news-systems/releases/tag/v1.1

**Comment by mjpost** [comment]
That works! Thanks for the quick response. It's quite large, though, with lots of files we don't need (particularly txt-ts).

Would you mind another option? I just need something I can `wget`. I think one of these might work:
* Create a branch, delete everything except `xml/`, and then create the release from that branch.
* TGZ up the `xml/` subdirectory and attach it as an asset to the release

**Comment by zouharvi** [comment]
Maintenance-wise I don't like the idea of having branches be ad-hoc subsets of each other.

However, it seems quite easy to just add extra assets to the release, which I did. Are these two links ok?
- https://github.com/wmt-conference/wmt24-news-systems/releases/download/v1.1/data.tar.gz
- https://github.com/wmt-conference/wmt24-news-systems/releases/download/v1.1/data_nots.tar.gz


The commands are:
```
tar czf data_nots.tar.gz humeval/esa_* humeval/mqm_* txt xml
tar czf data.tar.gz humeval/esa_* humeval/mqm_* txt xml txt-ts
```

**Comment by mjpost** [comment]
Agreed that this is an easier route, and I tested that it works. If you want to make it even smaller, however, I only need the XML directory (all the system outputs are in there, as well).

Thank you!

**Comment by zouharvi** [comment]
Ok, updated:

```
tar czf data.tar.gz humeval/esa_* humeval/mqm_* txt xml txt-ts
tar czf data_nots.tar.gz humeval/esa_* humeval/mqm_* txt xml
tar czf data_onlyxml.tar.gz xml
```

Which creates:

- https://github.com/wmt-conference/wmt24-news-systems/releases/download/v1.1/data.tar.gz
- https://github.com/wmt-conference/wmt24-news-systems/releases/download/v1.1/data_nots.tar.gz
- https://github.com/wmt-conference/wmt24-news-systems/releases/download/v1.1/data_onlyxml.tar.gz
